src/components/navbar/NavItem/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./styles.module.scss";
+
+interface Props {
+  label?: string;
+  onClick: () => void;
+  isActive?: Boolean;
```

Comment:
Boolean with a capital B is a function, make it "boolean" to avoid future issues.

Reply:
done